packages/hint-compat-api/src/core/deprecated-hint.ts

```diff
@@ -0,0 +1,38 @@
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { SimpleSupportStatement, VersionValue } from '../types-mdn.temp';
+
+import meta from '../meta/css';
+import { APIHint } from './api-hint';
+import { CSSFeatureStatus, CompatNamespace } from '../enums';
+import { FeatureInfo, BrowsersInfo } from '../types';
+import { Events, Event } from 'hint/dist/src/lib/types';
```

Comment:
IIRC, we have a convention (I don't know if it is write anywhere) for the `import`.

1st. Packages that node have by default (`fs`, `path`, etc.).
2nd. "External" dependencies (`hint`, `lodash`, etc.)
3rd. "Local" dependencies (`../types`, `../enum`, etc.)

This would apply to few files.

Reply:
I was not aware of this convention. However I was trying to apply a similar approach but it seems I missed some imports.

Reply:
Done.